templates/contests_view.html.twig

```diff
@@ -44,7 +44,9 @@
 <h3 id="index_pages">{{ msg('index-pages') }}</h3>
 <ol>
     {% for ip in contest.index_pages %}
-        <li><a href="{{ ip.url }}">{{ ip.url }}</a></li>
+    {# Usually, https://mul.wikisource.org/some_page redirects to https://wikisource.org/ #}
+    {# The replace filter ensures that the link contains the correct destination, as intended, while keeping the visible URL the same #}
+        <li><a href="{{ ip.url|replace({'https://mul.': 'https://'}) }}">{{ ip.url }}</a></li>
```

Comment:
I don't think this is the right place to be normalizing this URL. It should never be saved to the database in the first place with `mul.wikisource.org` as that's just a redirect to `wikisource.org`. So in IndexPageRepository when we call `$wikisource = $this->wikisourceApi->newWikisourceFromUrl( $indexPageUrl );` with a `mul` URL it should be getting back a normalized one. Then it'll work everywhere.

Reply:
Right, I thought perhaps we might still want to show `mul.wikisource.org/page` as the visible URL. 

I'll make the changes!

Reply:
Cool. Actually, I was thinking about this a bit more, and it seems like it's probably pretty unlikely that people will use the `mul.` form of the URLs anyway, so this is probably not a big issue.